httpserver/src/test/kotlin/tbdex/sdk/httpserver/handlers/SubmitRfqTest.kt

```diff
@@ -0,0 +1,70 @@
+package tbdex.sdk.httpserver.handlers
```

Comment:
super nit: maybe we could call this test class `TbdexHttpServerTest`? `SubmitRfqTest` says we're testing that class / that method, not starting up a server and sending a request w/ a client

Reply:
Do you think `SubmitRfqEndpointTest` is better? Tests for the other endpoints will not go in this file, so it's not really comprehensive of all `TbdexHttpServer` functionality